src/tools/dev/xml/GeneratePython.h

```diff
@@ -3309,7 +3307,14 @@ class PythonGeneratorAttribute : public GeneratorBase
         c << Endl;
         c << "    Py_DECREF(tuple);" << Endl;
         c << "    if( obj == NULL)" << Endl;
-        c << "        PyErr_Format(PyExc_RuntimeError, \"Unable to set unknown attribute: '\%s'\", name);" << Endl;
+        c << "        PyErr_Format(PyExc_RuntimeError, \"Unable to set attribute: '\%s'\", name);" << Endl;
```

Comment:
I needed to re-add this logic to handle handful of cases of `NULL` return from `_Set<FieldName>()` calls. However, if you look at, for example, color attribute vector stuff, that logic sets the error string using `PyErr_SetString(PyExc_IndexError, ...)` and returns `NULL` here but then here we call `PyErr_Format()` passing a new string. Does that wind up invalidating the string set earlier?